examples/hello_rust/hello_rust_main.rs

```diff
@@ -21,54 +21,115 @@
 /****************************************************************************
  * Attributes
  ****************************************************************************/
+ 
+ #![no_main]
+ #![no_std]
+ 
+ /****************************************************************************
+  * Uses
+  ****************************************************************************/
+ 
+ use core::panic::PanicInfo;
+ use core::ffi::{ c_char, c_int, c_void };
+ 
+ /****************************************************************************
+  * Externs
+  ****************************************************************************/
+ 
+ extern "C" {
+     pub fn printf(format: *const u8, ...) -> i32;
+     pub fn puts(s: *const c_char) -> c_int;
+     pub fn fgets(
+         buf: *mut c_char,
+         n: c_int,
+         stream: *mut c_void
+     ) -> *mut c_char;
+     pub fn lib_get_stream(fd: c_int) -> *mut c_void;
+ 
+     // Functions for Rust LED Blinky example
```

Comment:
```suggestion
     // Functions for Rust LED Blinky example
```

Remove the comment and the blank line above

Reply:
Okay.